### PR TITLE
feat: dp30 detail campaign

### PR DESCRIPTION
**Pull request by C1021G1datnvn**

### CODE REVIEW

File: src/components/Campaign/DetailCampaign.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { SoundFilled } from '@ant-design/icons';
+import { useParams } from 'react-router-dom';
+
+import './DetailCampaign.css';
+import { useFindOneCampaign } from '@hooks';
+import { DatePicker, Form, Input } from 'antd';
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: move top

File: src/components/Campaign/DetailCampaign.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import TextArea from 'antd/es/input/TextArea';
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: use Input.TextArea

File: src/components/Campaign/DetailCampaign.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import dayjs from 'dayjs';
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: move top

File: src/components/Campaign/DetailCampaign.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+export const DetailCampaign = () => {
+	const { id } = useParams();
+	const { data: detailCampaign, isLoading: detailCampaignLoading } =
+		useFindOneCampaign(+id!);
+	if (detailCampaignLoading) {
+		return null;
+	}
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: handle loading for page. not like this

File: src/components/Campaign/DetailCampaign.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+	return (
+		<div>
+			<div className="register_workspace">
+				<SoundFilled />
+				&nbsp;
+				<span> Update Campaign</span>
+			</div>
+			<hr />
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: use component of antd

File: src/components/Campaign/DetailCampaign.tsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+			<Form labelCol={{ span: 10 }} wrapperCol={{ span: 20 }}>
+				<Form.Item label="Name" name={'name'}>
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: why name={'name'} ?

File: src/components/Campaign/DetailCampaign.tsx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+					<Input
+						disabled
+						placeholder="Input name"
+						defaultValue={detailCampaign.name}
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: use initialValue for Form in antd. Not like this

File: src/components/Campaign/DetailCampaign.tsx [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+					/>
+				</Form.Item>
+
+				<Form.Item label="Description" name={'description'}>
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: same

File: src/components/Campaign/DetailCampaign.tsx [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+					<TextArea
+						disabled
+						placeholder="Input description"
+						defaultValue={detailCampaign.description}
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: same

File: src/components/Campaign/DetailCampaign.tsx [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+					/>
+				</Form.Item>
+
+				<Form.Item
+					className="timestampInitial"
+					label="Expired time"
+					name={'expired_time'}
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: same

File: src/components/Campaign/DetailCampaign.tsx [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+				>
+					<DatePicker
+						disabled
+						defaultValue={dayjs(detailCampaign.expired_time)}
```
Comment (stunited-thanhle) [inline src/components/Campaign/DetailCampaign.tsx]: same

**Review by demosofa — APPROVED** [review]

**Pull request merged**